spyder/config/main.py

```diff
@@ -573,7 +573,45 @@
               'zenburn/comment':    ('#7f9f7f', False, True),
               'zenburn/string':     ('#cc9393', False, False),
               'zenburn/number':     ('#8cd0d3', False, False),
-              'zenburn/instance':   ('#dcdccc', False, True)
+              'zenburn/instance':   ('#dcdccc', False, True),
+              # ---- SolarizedDark ----
+              'solarizeddark/name':        "SolarizedDark",

```

Comment:
The name is what appears in Spyder Preferences, so it needs to be `Solarzied Dark`